bin/update_config.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -ex
+# env vars: CLUSTER_NAME, CONFIG_BRANCH, CONFIG_REPO, NAMESPACE
+
+BIN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $BIN_DIR/../docker/bin/set_git_env_vars.sh # sets DOCKER_IMAGE_TAG
+
+pushd $(mktemp -d)
+git clone --depth=1 -b ${CONFIG_BRANCH:=master} ${CONFIG_REPO:=github-mozmar-robot:mozmeao/basket-config} basket-config
+cd basket-config
+
+set -u
+for DEPLOYMENT in {clock-,donateworker-,fxaeventsworker-,fxaworker-,web-,worker-,}deploy.yaml; do
```

Comment:
as we're passing in multiple clusters in `.gitlab-ci.yml`, we'll need to add an outer loop here as we do [in snippets](https://github.com/mozmeao/snippets-service/blob/master/bin/update-config.sh#L11)